src/mapping/config/MappingConfiguration.cpp

```diff
@@ -336,8 +339,8 @@ MappingConfiguration::MappingConfiguration(
   std::list<XMLTag> GaussRBF{
       XMLTag{*this, RBF_GAUSSIAN, once, SUBTAG_BASIS_FUNCTION}.setDocumentation("Gaussian basis function accepting a support radius or a shape parameter.")};
   attrShapeParam.setDefaultValue(std::numeric_limits<double>::quiet_NaN());
-  attrSupportRadius.setDefaultValue(std::numeric_limits<double>::quiet_NaN());
-  addAttributes(GaussRBF, {attrShapeParam, attrSupportRadius});
+  attrSupportRadius.setDefaultValue("NaN");
```

Comment:
The sentinel value should probably be `""` here and then use `.empty()` to figure out if an argument was passed or not.